content/03.methods.md

```diff
@@ -19,4 +19,13 @@ talk about the discriminator model and how it works
 talk about calibrating deep learning models with temperature smoothing
 
 ## Experimental Design
-talk about sampling experiment
+Being able to re-use label functions would significantly speed up the annotation process.
+To test the viability of re-use we implemented a sampling with replacement approach.
+For each relationship we trained a generative model using only label functions in the database category.
```

Comment:
"database category" seems unclear? Do you mean edge type / metaedge? Let's get a consistent terminology.